relay-server/src/services/processor/span/processing.rs

```diff
@@ -259,6 +281,18 @@ pub async fn process(
     }
 }
 
+/// Whether or not to convert spans into backward-compatible V2 spans.
+///
+/// This only makes sense when we forward the envelope to Kafka.
+fn produce_compat_spans(config: &Config, global_config: &GlobalConfig) -> bool {
+    #[cfg(not(feature = "processing"))]
+    {
+        return false;
+    }
+    config.processing_enabled()
+        && utils::sample(global_config.options.span_kafka_v2_sample_rate).is_keep()
```

Comment:
Can also do an org rollout, then we can make sure we test with Sentry only first.